content/posts/jellyfin-10.5.md

```diff
@@ -0,0 +1,121 @@
++++
+title = "Jellyfin 10.5"
+subtitle = "A step closer to the best experience with your media"
+author = "Fernando"
+githubusername = "ferferga"
+date = "2020-03-06"
+justify = "center"
++++
+
+Hello Jellyfin users out there! Welcome back to our blog!
+
+What would you think if I tell you that 10.5 is probably the biggest release we've done? Fasten your seat belts and get ready because the numbers are big:
+**Over 200 contributions from the community were merged and over 500 issues were fixed** since 10.4.3 (the latest stable version), both in Jellyfin and Jellyfin Web,
+the interface you access using your browser.
```

Comment:
I will leave a mix with the suggestions of others in the following commit. I'll leave the ``the interface you access using your browser.`` sentence, because newcomers might not know what it is just at the beginning.